google/cloud/pubsub_v1/subscriber/_protocol/streaming_pull_manager.py

```diff
@@ -106,9 +106,11 @@ class StreamingPullManager(object):
             ``projects/{project}/subscriptions/{subscription}``.
         flow_control (~google.cloud.pubsub_v1.types.FlowControl): The flow
             control settings.
-        use_legacy_flow_control (bool): Disables enforcing flow control settings
-            at the Cloud PubSub server and uses the less accurate method of only
-            enforcing flow control at the client side.
+        use_legacy_flow_control (bool):
```

Comment:
Perhaps? I prefer a bit of repetition with parallel construction to make things extra clear.

If set to ``True``, flow control at the Cloud Pub/Sub server is disabled, though client-side flow control is still enabled. If set to `False``, both server-side and client-side flow control are enabled.

Reply:
Reworded, more clarity is always good.